Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM scratch
 COPY script/ca-certificates.crt /etc/ssl/certs/
 COPY dist/traefik /
+ WORKDIR /tmp
```

Comment:
These two `WORKDIR` should be before the first `COPY` we will be able to use cache ;)